Automattic/wp-calypso
# Add new availability message.

#### Changes proposed in this Pull Request

* Update the error message shown when attempting purchase a mapping for a subdomain that has a pre-existing conflicting CNAME record.

#### Testing instructions

* Apply D46527-code
* Create a CNAME record for a domain that you own (ex. foo.example.com)
* Attempt to purchase a mapping for the subdomain
* Make sure that the correct error message is shown:
<img width="1069" alt="Screen_Shot_2020-07-17_at_1_34_09_PM" src="https://user-images.githubusercontent.com/1379730/87821637-260af000-c83e-11ea-8b0c-3c8c5fb51139.png">
